### PR TITLE
feat(snapshots): add delete snapshot grpc

### PR DESCRIPTION
**Pull request by Abhinandan-Purkait**

### CODE REVIEW

File: control-plane/grpc/proto/v1/volume/volume.proto [hunk 1]
```diff
@@ -403,6 +403,19 @@ message CreateSnapshotReply {
     common.ReplyError error = 2;
   }
 }
+
+// Delete a snapshot of the volume
+message DeleteSnapshotRequest {
+  // uuid of the volume
+  string volume_id = 1;
+  // uuid of the snapshot
+  string snapshot_id = 2;
```
Comment (tiagolobocastro) [inline control-plane/grpc/proto/v1/volume/volume.proto]:
```suggestion
  string    volume_id = 1;
  // uuid of the snapshot
  string snapshot_id = 2;
```

**Review by tiagolobocastro — APPROVED** [review]

**Review by dsharma-dc — APPROVED** [review]

**Comment by dsharma-dc** [comment]
nit-pick: You may want to modify commit heading to say `implement/provide "delete snapshot grpc"`
It's a bit confusing to read `add delete snapshot grpc`

**Comment by Abhinandan-Purkait** [comment]
> nit-pick: You may want to modify commit heading to say `implement/provide "delete snapshot grpc"` It's a bit confusing to read `add delete snapshot grpc`

Sure would change to that.

**Comment by Abhinandan-Purkait** [comment]
bors merge

**Pull request merged**